scripts/vagrant/setup-counter-processor.sh

```diff
@@ -11,7 +11,7 @@ python3.6 -m ensurepip
 COUNTER_USER=counter
```

Comment:
Whoops, I missed this when making a review:

> Special notes for your reviewer: The vagrant script references the non-working GeoLite db download URL. I think it is currently failing gracefully but this section could perhaps be removed. Since the new process requires having an account/signing a EULA with MaxMind, probably configuring a cron job to run their update script, etc. I'm not sure it is something we could/should automate. So it may make sense to do something like write a message to say 'follow the guides to install the optional GeoLite Country db for counter-processor to capture per country MakeDataCount statistics.' I'm hoping someone might suggest a change for the script.

I think it would be fine to comment out this part of the Vagrant script. Perhaps with a comment above that section saying "This can no longer be automated. See the Installation Guide."

Reply:
I see the lines were removed and a note added. Great!